### ISSUE TITLE
parallel mistnet runs are slow

### ISSUE THREAD

**Issue by adokter** [adokter/vol2birdR#16]
When calling vol2birdR from bioRad, a parallel run takes longer than a serial run:
```
process_file <- function(file_in, mistnet){
  # run calculate_vp()
  vp <- calculate_vp(file_in, mistnet = mistnet)
}

library(parallel)
mclapply(my_pvolfiles, process_file, mc.cores = 10, mistnet = TRUE)
```
Here `my_pvolfiles` is a vector with filenames

Disabling mistnet does speed up the run compared to a serial run:
```
mclapply(my_pvolfiles, process_file, mc.cores = 10, mistnet = FALSE)
```

**Comment by adokter** [comment]
@andershenja any idea why?

**Comment by adokter** [comment]
closed by [0e0e7e8](https://github.com/adokter/vol2birdR/commit/0e0e7e86ac44561baebb73dea1ca0c7f9bf6d773)
